nifty/pipeline/objectoriented/GetConfig.py

```diff
@@ -121,3 +124,14 @@ def makeConfig(self):
             with open('./' + self.configFile, 'w') as self.outfile:
                 self.config.write(self.outfile)
             logging.info("\nData reduction parameters for this reduction were copied from recipes/defaultConfig.cfg to ./config.cfg.")
+
+        if self.cadc:
+            try:
+                with open('./' + self.configFile, 'r') as self.config_file:
```

Comment:
What happens here if the config file doesn't already exist?

Reply:
See 74a11e9 for better error handling.